test/visualization/CMakeLists.txt

```diff
@@ -8,6 +8,8 @@ set(build TRUE)
 PCL_SUBSYS_OPTION(build "${SUBSYS_NAME}" "${SUBSYS_DESC}" ${DEFAULT} "${REASON}")
 PCL_SUBSYS_DEPEND(build "${SUBSYS_NAME}" DEPS ${SUBSYS_DEPS} OPT_DEPS ${OPT_DEPS})
 
+include_directories(${VTK_INCLUDE_DIRS})
```

Comment:
Yeah, for some reason my test was not compiling, but there's no reason to add this in this PR.